### PR TITLE
Added python 2.7 to compilation requirements.

### PR DESCRIPTION
**Pull request by Jaxkr**

### CODE REVIEW

**Comment by webmaster128** [comment]
Hey @Jaxkr, thank you for your contribution!

I am wondering why Python 2.7 is required. Where are we using that?

**Comment by Jaxkr** [comment]
Hi, I am not sure where you're using it, but the command:

```./configure --disable-documentation --prefix=$PWD/../libpqxx-installation && make -j 4 install
```

fails if the python command does not exist on the current PATH.

**Comment by Jaxkr** [comment]
Additionally, I am having some trouble compiling using the provided instructions.

When I run the cmake command I get the following output:
https://pastebin.com/raw/e1FNqPt5

**Comment by webmaster128** [comment]
I found a recent commit message that indicates that Python 3 is supported as well: https://github.com/jtv/libpqxx/commit/1ef5738173746bbdd377b92588e1195410c535aa. Could you rephrase that part somehow, such that people on Python3-only systems do not install Python 2.7?

**Comment by Jaxkr** [comment]
@webmaster128 Changed. :)

**Comment by webmaster128** [comment]
I just added a license to this project (MIT). Please let me know @Jaxkr if you agree to distribute your contributions under this license. Then I can merge this. Thanks again!

Btw. did you see my update in #6? I guess you can compile now

**Comment by Jaxkr** [comment]
@webmaster128 Of course! :)
Feel free to merge it, thanks for supporting this project.
